### ISSUE TITLE
WildStacker Stacked Spawners not counted

### ISSUE THREAD

**Issue by MCLimited** [BentoBoxWorld/Level#255]
### Expected behavior

With the use of Wildstacker its possible to stack spawners (for example stack 20 spawners into 1 spawnerblock).

When there are 20 spawners I would expect that it should be counted 20 times for the island level since its stacked 20 times.

In this picture I placed 1 spawner. From this point I need 50 more point for the next level
![50 points](https://user-images.githubusercontent.com/18746382/151610276-013daf44-acd8-4232-90b7-b4f0f293e7d0.png)

### Observed/Actual behavior

My obervation was that when I stacked the spawner (see picture) that I still need 50 points for the next level instead of 49 points (keeping in mind that 1 spawner is worth 1 point in the point system).
![50 points with 2 spawners](https://user-images.githubusercontent.com/18746382/151610239-0ba2a8b3-0f5f-40cc-8153-1a97ce0a718e.png)

Unfortunately its not counting the stacked blocks. 

### Steps/models to reproduce

Using the same addons/plugins as described below should reproduce the problem.

### BentoBox version

[19:35:42 INFO]: Running PAPER 1.18.1.
[19:35:42 INFO]: BentoBox version: 1.20.0-SNAPSHOT-b2221
[19:35:42 INFO]: Database: JSON
[19:35:42 INFO]: Loaded Game Worlds:
[19:35:42 INFO]: skyblock (BSkyBlock): Overworld, Nether, The End
[19:35:42 INFO]: Loaded Addons:
[19:35:42 INFO]: Bank 1.4.0 (ENABLED)
[19:35:42 INFO]: Biomes 1.14.0 (ENABLED)
[19:35:42 INFO]: BSkyBlock 1.15.2 (ENABLED)
[19:35:42 INFO]: Challenges 1.0.0-SNAPSHOT-b475 (ENABLED)
[19:35:42 INFO]: ControlPanel 1.13.0 (ENABLED)
[19:35:42 INFO]: DimensionalTrees 1.6.0 (ENABLED)
[19:35:42 INFO]: Level 2.9.0-SNAPSHOT-b494 (ENABLED)
[19:35:42 INFO]: Limits 1.19.0 (ENABLED)
[19:35:42 INFO]: MagicCobblestoneGenerator 2.3.0 (ENABLED)
[19:35:42 INFO]: Warps 1.11.2 (ENABLED)
[19:35:42 INFO]: WildStackerHook 1.2.0-SNAPSHOT-b9 (ENABLED)

### Plugin list

- BentoBox
- CMI
- CMILib
- DeluxeChat
- LuckPerms
- PlaceholderAPI
- PlugMan
- TAB-BukkitBridge
- Vault
- WildStacker
- WorldEdit
- WorldGuard
- XConomy

### Other

I tried using rosestacker and resetting/different versions of de levels addon as well. I had unfortunately the same results.

**Comment by tastybento** [comment]
Yes, the hook for Wild Stackers only supported blocks and not spawners. I've had a go at adding support but I don't have a test setup to try it. So let me know if 2.9.1-SNAPSHOT from https://ci.codemc.io/job/BentoBoxWorld/job/Level/ works for you or not. If not, post the error. Thanks.

**Comment by MCLimited** [comment]
Good evening @tastybento.

Thanks for the quick support. Its working perfect now. The first spawner only counts as 2 blocks but that is not a big deal. Stacking is now working!

I needed 50 more points (no spawners placed)
I needed 48 more points (1 spawner stacked (1spawner block))
I needed 40 more points (9 spawners stacked (1spawner block))
I needed 10 more points (39 spawners stacked (1spawner block))

its only spamming the console with the following message:
WARN]: Skipping BlockEntity with id 

But also this is not a big deal since its calculating the stacked spawner blocks now :D

Thank you!
